politeiawww/api/cms/v1/v1.go

```diff
@@ -510,7 +510,7 @@ type LineItemPayouts struct {
 // LineItemPayoutsReply returns an array of line items within the requested
 // date range.
 type LineItemPayoutsReply struct {
```

Comment:
Should this route be renamed since its returning invoices now and now line items?

Reply:
Yah decent point.  Any ideas on name?  InvoicePayoutsReply?

Reply:
Yeah, I think `InvoicePayouts` is fine.  You could also do `GetInvoices` if you wanted to be consistent with `GetDCCs`.